src/main/resources/org/sagebionetworks/web/client/widget/entity/EntityBadgeViewImpl.ui.xml

```diff
@@ -13,7 +13,7 @@
 					<b:Icon ui:field="icon" type="FILE" />
 				</g:FocusPanel>
 			</t:TableData>
-			<t:TableData ui:field="entityTableData">
+			<t:TableData ui:field="entityTableData" paddingTop="6" >
```

Comment:
Small correction for the File Browser UI (lots of problems here - the content height of these elements are not used since these are inline)